src/core/column/nth.h

```diff
@@ -23,54 +23,65 @@
 #define dt_NTH_h
 #include "column/virtual.h"
 #include "stype.h"
-
 namespace dt {
+
+
 template<typename T, bool SKIPNA>
-class NTH_ColumnImpl : public Virtual_ColumnImpl {
+class Nth_ColumnImpl : public Virtual_ColumnImpl {
   private:
     Column col_;
-    int32_t n_;
     Groupby gby_;
+    int32_t n_;
```

Comment:
Any particular reason for the positional argument change between gby and n? Again, part of me learning

Reply:
Just refactoring. Since groupby is related to the column, it is better to keep these two together. Also, in this case the padding (that is related to the usage ot `int32_t`) goes immediately after the `n_` declaration. If, in future, we add any other parameters at the end, we will easily see how much padding we should add.